packages/frontend/src/components/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+interface TooltipProps {
+  children: React.ReactNode
```

Comment:
Maybe we can change this type to the below one, to always require usage of both components
```suggestion
  children: [
    ReactElement<TooltipTriggerProps>,
    ReactElement<TooltipContentProps>,
  ]
```

Reply:
goood stuuuuf